src/components/scrollable-canvas/scrollable-canvas.jsx

```diff
@@ -12,26 +12,34 @@ const ScrollableCanvasComponent = props => (
         )}
     >
         {props.children}
-        <div className={styles.horizontalScrollbarWrapper}>
+        <div
+            className={styles.horizontalScrollbarWrapper}
+            style={{'pointer-events': 'none'}}
+        >
             <div
                 className={styles.horizontalScrollbar}
                 style={{
-                    width: `${props.horizontalScrollLengthPercent}%`,
-                    left: `${props.horizontalScrollStartPercent}%`,
-                    visibility: `${props.hideCursor ||
-                        Math.abs(props.horizontalScrollLengthPercent - 100) < 1e-8 ? 'hidden' : 'visible'}`
+                    'width': `${props.horizontalScrollLengthPercent}%`,
+                    'left': `${props.horizontalScrollStartPercent}%`,
+                    'pointer-events': 'all',
```

Comment:
The [MDN spec](https://developer.mozilla.org/en-US/docs/Web/CSS/pointer-events) for `pointer-events` says that 'all' only applies to svg content, but this style is getting applied to a `div`. Should the value be `auto` instead?

Reply:
Thanks!